client.go

```diff
@@ -626,8 +626,16 @@ func (s *Client) LoopInQuote(ctx context.Context,
 	//
 	// TODO(guggero): Thread through error code from lnd to avoid string
 	// matching.
-	minerFee, err := s.lndServices.Client.EstimateFeeToP2WSH(
-		ctx, request.Amount, request.HtlcConfTarget,
+	witnessProg := [32]byte{}
+	dummyAddr, err := btcutil.NewAddressWitnessScriptHash(
+		witnessProg[:], s.lndServices.ChainParams,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	minerFee, err := s.lndServices.Client.EstimateFee(
```

Comment:
Can use the helper method `EstimateFeeToP2WSH` that was added back in `lndclient v0.15.0-6`.